webpack.config.js

```diff
@@ -7,7 +7,7 @@ const TerserWebpackPlugin = require('terser-webpack-plugin');
 const CaseSensitivePathsPlugin = require('case-sensitive-paths-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
-const getCSSModulesLoadersConfig = isDevMode => {
+const getCSSModulesLoadersConfig = () => {
```

Comment:
It might be more clear to inline this into `commonConfig` alongside the `.tsx?` rule and remove the separate `devModules`/`prodModules` intermediates, now that it isn't devMode-dependent.